### PR TITLE
Update README to have details on password rotation

### PR DESCRIPTION
**Pull request by MiaAltieri**

### CODE REVIEW

**Review by welpaolo — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -68,6 +68,24 @@ We have also added support for the database legacy relation from the [original v
 juju relate mongodb graylog
 ```
 
+### Password rotation
+#### Internal admin user
+The admin user is used internally by the Charmed MongoDB Operator, the `set-admin-password` action can be used to rotate its' password.
```
Comment (zmraul) [inline README.md:73]:
```suggestion
The admin user is used internally by the Charmed MongoDB Operator, the `set-admin-password` action can be used to rotate its password.
```

File: README.md [hunk 2]
```diff
@@ -68,6 +68,24 @@ We have also added support for the database legacy relation from the [original v
...
+```shell
+# to set a specific password for the admin user
+juju run-action mongodb/<leader unit> set-admin-password password=<password> --wait
```
Comment (zmraul) [inline README.md:76]: You can actually do:
```suggestion
juju run-action mongodb/leader set-admin-password password=<password> --wait
```

**Review by zmraul — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -68,6 +68,24 @@ We have also added support for the database legacy relation from the [original v
...
 
+### Password rotation
+#### Internal admin user
+The admin user is used internally by the Charmed MongoDB Operator, the `set-admin-password` action can be used to rotate its password.
+```shell
+# to set a specific password for the admin user
+juju run-action mongodb/leader set-admin-password password=<password> --wait
+
+# to randomly generate a password for the admin user
+juju run-action mongodb/<leader unit> set-admin-password --wait
```
Comment (zmraul) [inline README.md:79]:
```suggestion
juju run-action mongodb/leader set-admin-password --wait
```

**Review by zmraul — APPROVED** [review]

**Pull request merged**